packages/hoppscotch-common/src/components/app/Banner.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div
+    class="relative flex items-center px-4 py-2 transition text-tiny group"
+    :class="bannerColor"
+  >
+    <icon-lucide-info class="mr-2 text-white" />
```

Comment:
Shouldn't we use a different icon based on the banner type? We might need to determine a color that fits in well with the background.

Reply:
Do we need to have a different icons for different types of the banner ( error, warning, info )? In my opinion, I feel the info icon conveys the meaning as a banner is typically used to show important information irrespective of what the banner is being used for. However, I am open to your idea if its better and I can maybe add a new property to the banner where we can define custom icons or specific icons which map to the respective type of the banner.

Reply:
I have selected darker colours for the background of the banner, so that the 	text-white	 color for the icon is displayed perfectly. Open to suggestions..

Reply:
It can be subjective, but I've seen the `info` icon often associated with informational messages. For instance, [Vuetify](https://vuetifyjs.com/en/components/alerts/#type), [Material UI](https://mui.com/material-ui/react-alert/#description).

I'd love to hear what others think. Let's maybe revisit it when the use case comes in the future.

Reply:
What icons would you prefer for error and warning banner type if we might go ahead with the suggestion?

Reply:
We use `info` icon. https://lucide.dev/icons/info

Reply:
@jamesgeorge007 @JoelJacobStephen

Reply:
Okay. I will use these icons for the specific icon types.

Reply:
@liyasthomas @jamesgeorge007 

These are the colors and icons used for each of the banner types:

1. Info Banner
<img width="988" alt="image" src="https://github.com/hoppscotch/hoppscotch/assets/70131076/7e18081a-2417-48ff-af95-8cb513387606">

2. Warning Banner
<img width="995" alt="image" src="https://github.com/hoppscotch/hoppscotch/assets/70131076/44a155e1-85fc-4f56-9c0c-cd9f87a7306f">

3. Error Banner
<img width="988" alt="image" src="https://github.com/hoppscotch/hoppscotch/assets/70131076/74982f1e-b82c-4889-9756-2c009cf075ff">

Reply:
Looks good to me.